### PR TITLE
Honesty update

### PR DESCRIPTION
**Pull request by Nyco**
Now that we have a clearer vision of 2.1.x because we released 2.1.0-beta1, we can add/remove items with much more confidence

### CODE REVIEW

File: doc/Roadmap.md [hunk 1]
```diff
@@ -7,18 +7,17 @@ Please feel free to comment!
 * Push notifications:
     * [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html)
     * Amazon AWS SNS (Simple Notification Service)
-* Instant Stream Resumption
 * Full text search in archive
-* Bind 2.0 (experimental)
```
Comment (michalwski) [inline doc/Roadmap.md]: This is still valid. Bind 2.0 is not in beta1, it doesn't mean it will not fit into final 2.1

Comment (Nyco) [inline doc/Roadmap.md]: Ok, once again my twisted mind that refuses to understand that. Fixed.

File: doc/Roadmap.md [hunk 2]
```diff
@@ -7,18 +7,19 @@ Please feel free to comment!
 * Push notifications:
     * [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html)
     * Amazon AWS SNS (Simple Notification Service)
-* Instant Stream Resumption
 * Full text search in archive
+* Experimental pipelining
+* Instant Stream Resumption
```
Comment (michalwski) [inline doc/Roadmap.md]: The pipelining we have doesn't equal `Instant Stream Resumption` as in the proto-XEP.

Comment (michalwski) [inline doc/Roadmap.md]: But we can still try to deliver this, as soon as new `ISR` is accepted as draft which I see possible.

**Pull request merged**